src/olympia/blocklist/mlbf.py

```diff
@@ -77,7 +77,7 @@ def generate_mlbf(stats, include, exclude):
 # Extends the BlockType enum to include versions that have no block of any type
 MLBFDataType = Enum(
     'MLBFDataType',
-    [block_type.name for block_type in BlockType] + ['NOT_BLOCKED', 'not_blocked'],
```

Comment:
This is the primary fix to the cause of https://github.com/mozilla/addons/issues/15261#issuecomment-2584947155